src/cli.ts

```diff
@@ -115,12 +115,21 @@ async function main() {
   let output = `<!doctype html>
 <title>${title}</title>
 <style>
+html, body {
+  height: 100%;
```

Comment:
The reason I hadn't done this in the past is that it doesn't resize on window resize.  But I guess it's better than nothing.

Reply:
I pushed a commit to track resizes.